launch_testing/launch_testing/asserts/assert_output.py

```diff
@@ -91,3 +103,91 @@ def assertInStdout(proc_output,
         assert False, "Did not find '{}' in output for any of the matching processes: {}".format(
             expected_output, names
         )
+
+
+def assertInStdout(proc_output,
+                   expected_output,
+                   process,
+                   cmd_args=None,
+                   *,
+                   output_filter=None,
+                   strict_proc_matching=True,
+                   strip_ansi_escape_sequences=True):
+    """
+    Assert that 'output' was found in the standard output of a process.
```

Comment:
@clalancette meta: I wonder if we should refer to the base function to avoid replicating docs.

Reply:
Yeah, I wasn't quite sure how to go about this.  I think I'll do what you suggest and just point the documentation to `assertInStream`.